### ISSUE TITLE
Should the text alternative for a video without audio always be visually present?

### ISSUE THREAD

**Issue by 2biazdk** [act-rules/act-rules.github.io#1815]
I have a video (no audio) that provides a series of short clips to set an atmosphere. Just as if this had been an image, I have made sure to provide a nice alternative text in the form of a paragraph that is visually hidden, so that only tools such as screen readers and search engine crawlers can pick it up.

These ACT Rules seem to imply that my alternative text must be visually present on the page to all users:
https://act-rules.github.io/rules/c3232f
https://act-rules.github.io/rules/ee13b5

But as with images, alternative text is typically not visually presented. Also, I do not see in WCAG a requirement for this to be visual when it describes a video without audio: https://www.w3.org/WAI/WCAG21/Techniques/general/G159

It would be interesting to open a discussion about whether such a video alternative text really _must_ be visually presented, or whether the ACT Rules should be updated to reflect that it _does not_ need to be presented visually.

**Comment by Jym77** [comment]
CG resolution: remove the visibility condition, plus add an assumption that sighted users can understad the video.

**Comment by bruce-usab** [comment]
Just to throw a wrench into the works, I would argue that video-only media which only sets an atmosphere meets the definition for [pure decoration](https://www.w3.org/TR/WCAG22/#dfn-pure-decoration).   If you agree, there is no requirement for a text alternative under 1.2.1 and consequently the invisible SR-only text is fine.

**Comment by WilcoFiers** [comment]
This one surprised me. I very much thought this was required, but thinking through it carefully, I think there is one pretty glaring problem with this; Animated gifs. If 1.2.1 required visible alternatives, then 1. either animated gifs need a visible alternative, or 2. something about animated gifs would make not quality as "video-only" content.

1. It seems generally acceptable that animated gifs can pass WCAG with an alt attribute. I also can't really think of what the benefit for someone with a disability would be. Granted there is the case where some people struggle to process things like social cues from images, but that's as true for still images as it is for short video clips, and it isn't required for still images. So that doesn't seem right.

2. Maybe animated gifs aren't "video-only" content. Looking at the definition of video-only I can't think of an argument for why that might be true. They are a "time-based presentation that contains only video", with video being "the technology of moving or sequenced pictures or images".

**Comment by bruce-usab** [comment]
Animated GIFs also came to my mind as an example of video-only content, and that ALT can be sufficient for those.

From [SC 1.1.1](https://www.w3.org/TR/WCAG22/#time-based-media):
> If non-text content is time-based media, then text alternatives at least provide descriptive identification of the non-text content. (Refer to Guideline 1.2 for additional requirements for media.)

It follows that if ALT is a "text alternative that serves the equivalent purpose" (and not merely providing descriptive identification) then 1.2 is not applicable.

**Comment by bruce-usab** [comment]
@WilcoFiers please check if I accidently deleted your comment!  I was agreeing with your two points, but I do not see them now.

Edit:  Wilco's [comment is in wcag thread](https://github.com/w3c/wcag/issues/3642#issuecomment-1920004271).

**Comment by Jym77** [comment]
For gifs and `alt`, I think this is covered by allowing the alternative to **not** be visible (whether it's a `sr-only` text or an `alt` attribute is pretty much the same).

The issue I see with considering "atmosphere" videos as "pure decoration" is that videos should be pausable (under 2.2.2 Pause, Stop, Hide). Which implies having some "pause" button, which should be accessible. And I think it would be confusing to expose a "pause video" button without exposing any video.
Or, said otherwise, what HTML/JS would you use to mark a video as "pure decoration" (and hide it), while still passing 2.2.2?

**Comment by dbjorge** [comment]
> It follows that if ALT is a "text alternative that serves the equivalent purpose" (and not merely providing descriptive identification) then 1.2 is not applicable.

I don't think this follows at all - there is nothing in 1.2 that says that it's only applicable if you meet a 1.1.1 in a particular way.

In general, I'll repeat-by-reference what I wrote in [this comment in w3c/wcag#3642](https://github.com/w3c/wcag/issues/3642#issuecomment-1919814253): I think we should be careful when discussing this to distinguish between the "text alternative" required by 1.1.1 and the "alternative to time-based media" required by 1.2.1. They are separately defined concepts with substantially different requirements.

(I'd encourage us to resolve the w3c/wcag thread before continuing here, I think it'll be confusing to have such overlapping discussion in multiple places)

**Comment by bruce-usab** [comment]
Resolving w3c/wcag#3642 before continuing here is sensible.  I do want to agree with @Jym77 that 2.2.2 is quite relevant.  @2biazdk you said <q>short clips</q> and I assumed that meant less than five seconds.  But that was presumptuous on my part.

**Comment by kengdoj** [comment]
https://github.com/w3c/wcag/issues/3642 has been resolved: 

No, SC 1.2.1 does not require alternative for time-based media for video-only to be visible.

**Comment by bruce-usab** [comment]
@kengdoj can this issue be closed?

**Comment by kengdoj** [comment]
@bruce-usab there's some activity on https://github.com/w3c/wcag/issues/3642#issuecomment-4576412718 so not sure this should be closed (and this issue isn't mine)
